src/libraries/System.IO.FileSystem.AccessControl/src/System/Security/AccessControl/FileSystemSecurity.cs

```diff
@@ -14,23 +14,23 @@ public abstract class FileSystemSecurity : NativeObjectSecurity
         private const ResourceType s_ResourceType = ResourceType.FileObject;
 
         internal FileSystemSecurity(bool isContainer)
-            : base(isContainer, s_ResourceType, _HandleErrorCode, isContainer)
+            : base(isContainer, s_ResourceType, _HandleErrorCode!, isContainer)
```

Comment:
I think I need to update the delegate return value into nullable, then this bang will no needed PR for that https://github.com/dotnet/runtime/pull/31781: